article.go

```diff
@@ -178,13 +178,18 @@ func (a *Article) Data() ([]byte, error) {
 
 		var blob []byte
 		var ok bool
+		var dec io.ReadCloser
 		if blob, ok = blobLookup(); !ok {
 			b, err := a.z.bytesRangeAt(start+1, end+1)
 			if err != nil {
 				return nil, err
 			}
 			bbuf := bytes.NewBuffer(b)
-			dec, err := NewXZReader(bbuf)
+			if compression == 5 {
```

Comment:
could you make a switch here so it will help for readability and future compression?